client/blocks/reader-full-post/featured-image.jsx

```diff
@@ -4,15 +4,18 @@
 import React from 'react';
 
 export default class FeaturedImage extends React.Component {
```

Comment:
I think so. Even though the post normalizer is taking care of it, this gives us a deeper defense against weird things that crop up.